terraform/install.sh

```diff
@@ -236,7 +255,7 @@ log "Checking Prerequisites..."
 getBillingAccount;
 
 log "Install current version of Terraform"
-installTerraform
+#installTerraform
```

Comment:
You can remove this given that now it's part of the Cloud Shell container image

Reply:
Ok.

Reply:
@simonz130 aren't we leaving the cloud shell container out for this release though?

Reply:
No, we are adding cloud shell custom image in upcoming release - the Cloud Shell team did fix the blocker on their side.